test/analysis/flux_variability_analysis.jl

```diff
@@ -79,6 +79,6 @@ end
         ],
     )
 
-    @test isapprox(maxs["EX_ac_e"]["EX_ac_e"], 16.81521027177164, atol = TEST_TOLERANCE)
-    @test isapprox(mins["EX_ac_e"]["EX_ac_e"], 1.553675043862341e-10, atol = TEST_TOLERANCE)
+    @test isapprox(maxs["EX_ac_e"]["EX_ac_e"], 8.5185494, atol = TEST_TOLERANCE)
```

Comment:
wow, this is a big difference - why?

Reply:
that's because the per-reaction optimizations were run on a non-modified model; now the FBA and FVA models are the same as it should be.